appveyor.yml

```diff
@@ -1,34 +1,44 @@
-# Fix line endings on Windows
-init:
-  - git config --global core.autocrlf true
```

Comment:
Also, forgot to say @vladikoff, I have already removed this in all contrib plugins I have push rights, and things work fine.

My point is we shouldn't rely on this, that is all.